nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/BaseTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+
+package org.nuxeo.rest.management;
+
+import javax.inject.Inject;
+
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.core.api.impl.blob.JSONBlob;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.jaxrs.test.HttpClientTestRule;
+import org.nuxeo.jaxrs.test.HttpClientTestRule.Builder;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.ServletContainerFeature;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features(ManagementFeature.class)
+public class BaseTest {
```

Comment:
Can you please rename it to `ManagementBaseTest` to avoid the confusion with the existing `BaseTest` class?

Reply:
You also need to make this class `abstract`, otherwise mvn will complain about "No runnable methods".

Reply:
👍